src/types/index.ts

```diff
@@ -96,22 +99,36 @@ export type StepWithData = StepBase & {
 };
 
 export type QuestionState = {
+  /** The number of available points to display in the footer for the question. */
   available_points: AvailablePoints;
+  /** A boolean that will display the Continue/Next button when true and canAnswer is false */
   is_completed: boolean;
+  /** A number ID of the user's selected Answer */
   answer_id?: number;
+  /** An array of Answer IDs that specify the order to display them when hasMultipleAttempts is true. */
   answer_id_order: ID[];
+  /** A string of the user's written response, used in two-step/WRM */
   free_response: string;
-  feedback_html: string; // Really only incorrect answer
+  /** A string of HTML to display below the incorrect answer */
+  feedback_html: string;
+  /** An ID of the answer to mark as correct. is_complete must also be true for this to display. */
   correct_answer_id: ID;
+  /** A string of HTML to display below the correct answer */
   correct_answer_feedback_html: string;
   response_validation?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
-  formats?: string[];
```

Comment:
I'm removing this from `questionsStates` because we aren't using it that way (where formats are used, it's checking formats on the exercise questions directly)